translator/src/test/java/org/finos/calmtranslator/translators/TestK8sModelTranslatorShould.java

```diff
@@ -1,26 +1,37 @@
 package org.finos.calmtranslator.translators;
 
-import java.io.IOException;
-
 import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.IOException;
 import org.finos.calmtranslator.calm.Core;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
 class TestK8sModelTranslatorShould {
 
-	private K8sModelTranslator translator;
-	private Core traderxCalmModel;
+    private K8sModelTranslator translator;
+    private Core traderxCalmModel;
+
+    @BeforeEach
+    void setUp() throws IOException {
+        this.translator = new K8sModelTranslator();
+        this.traderxCalmModel = new ObjectMapper().readValue(this.getClass().getClassLoader().getResourceAsStream("traderx-calm.json"), Core.class);
+    }
 
-	@BeforeEach
-	void setUp() throws IOException {
-		this.translator = new K8sModelTranslator();
-		this.traderxCalmModel = new ObjectMapper().readValue(this.getClass().getClassLoader().getResourceAsStream("traderx-calm.json"), Core.class);
-	}
+    @Test
+    void generateServices() {
+        String k8sManifest =  translator.translate(traderxCalmModel);
+        
+        //Simple namespace generation check
+        assertThat(k8sManifest).contains("Namespace");
+        assertThat(k8sManifest).contains("traderx-system");
```

Comment:
For the test can we compare the against a whole yaml? 

I think this test is a bit weak at present

Reply:
I did think about that. The suggestion means our tests need to be aware of the k8s api. When that changes (most likely via the fabric8 lib) we will need to adjust the k8s sample/test manifest.

This led me to think about testing using a much smaller calm model - which then means maintaining additional calm models in our tests.

What do you think? Both sound like a maintainance nightmare

Reply:
aha - using another object will make this nicer - updating coming soon

Reply:
@Budlee this looked a lot nice now - love to get your feedback :)